contracts/zero-ex/tests/utils/DeployZeroEx.sol

```diff
@@ -0,0 +1,304 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2022 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "forge-std/Test.sol";
+
+import "src/IZeroEx.sol";
+import "src/ZeroEx.sol";
+import "src/migrations/InitialMigration.sol";
+import "src/features/OwnableFeature.sol";
+import "src/features/SimpleFunctionRegistryFeature.sol";
+import "src/features/NativeOrdersFeature.sol";
+import "src/features/BatchFillNativeOrdersFeature.sol";
+import "src/features/FundRecoveryFeature.sol";
+import "src/features/TransformERC20Feature.sol";
+import "src/features/OtcOrdersFeature.sol";
+import "src/features/MetaTransactionsFeature.sol";
+import "src/features/nft_orders/ERC1155OrdersFeature.sol";
+import "src/features/nft_orders/ERC721OrdersFeature.sol";
+import "src/features/UniswapFeature.sol";
+import "src/features/multiplex/MultiplexFeature.sol";
+import "src/external/TransformerDeployer.sol";
+import "src/external/FeeCollectorController.sol";
+import "src/transformers/WethTransformer.sol";
+import "src/transformers/FillQuoteTransformer.sol";
+import "src/transformers/PayTakerTransformer.sol";
+import "src/transformers/AffiliateFeeTransformer.sol";
+import "src/transformers/PositiveSlippageFeeTransformer.sol";
+import "src/transformers/bridges/IBridgeAdapter.sol";
+import "src/transformers/bridges/EthereumBridgeAdapter.sol";
+
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/WETH9V06.sol";
+
+contract DeployZeroEx is Test {
+    ZeroEx public ZERO_EX = ZeroEx(0xDef1C0ded9bec7F1a1670819833240f027b25EfF);
+    IZeroEx public IZERO_EX = IZeroEx(address(ZERO_EX));
```

Comment:
This variable name is confusing. Typically objects starting with `I` are themselves interfaces, not contracts. Is `IZeroEx` meaningfully different from `ZeroEx`?